src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger/config.jelly

```diff
@@ -113,6 +115,14 @@
                                default=""
                                checkUrl="'descriptorByName/com.sonyericsson.hudson.plugins.gerrit.trigger.hudsontrigger.GerritTrigger/emptyOrIntegerCheck?value='+escape(this.value)"/>
                 </f:entry>
+                <f:entry title="${%Aborted}"
+                         field="gerritBuildAbortedVerifiedValue"
+                         help="/plugin/gerrit-trigger/trigger/help-GerritBuildAbortedVerified.html">
+                    <f:textbox name="gerritBuildAbortedVerifiedValue"
+                               value="${it.gerritBuildAbortedVerifiedValue}"
+                               default=""
+                               checkUrl="'descriptorByName/com.sonyericsson.hudson.plugins.gerrit.trigger.hudsontrigger.GerritTrigger/emptyOrIntegerCheck?value='+escape(this.value)"/>
```

Comment:
Shouldn't it be 
```suggestion
                               checkUrl="'descriptorByName/com.sonyericsson.hudson.plugins.gerrit.trigger.hudsontrigger.GerritTriggerDescriptor/emptyOrIntegerCheck?value='+escape(this.value)"/>
```
?
Have you verified that all checks and list fills etc are working after the extraction of the descriptor to a separate class?

Reply:
No, I did not do this and will perform additional testing in this area

Reply:
No, I tested the changes and the verifiers still work as previously. The descriptorByName will still calls the static getDescriptor method from GerritTrigger hence no change is needed